src/peer-store/address-book.js

```diff
@@ -183,6 +185,12 @@ class AddressBook extends Book {
     return multiaddrInfos
   }
 
+  _setPeerId (peerId) {
+    if (!this._ps.peerIds.get(peerId)) {
+      this._ps.peerIds.set(peerId.toString(), peerId)
```

Comment:
Use B58 string for consistency.
```suggestion
      this._ps.peerIds.set(peerId.toB58String(), peerId)
```